R/get_lymphgen.R

```diff
@@ -7,169 +7,266 @@
 #'
 #' @param flavour Lymphgen flavour.
 #' @param these_samples_metadata A metadata table to auto-subset the data to samples in that table before returning.
-#' @param return_feature_matrix Boolean parameter, default is FALSE.
-#' @param return_feature_annotation Boolean parameter, default is FALSE.
 #' @param lymphgen_file Path to lymphgen file.
 #' @param keep_all_rows Boolean parameter, default is FALSE.
 #' @param keep_original_columns Boolean parameter, default is FALSE.
+#' @param streamlined Boolean, set to true to get just a data frame with one column for sample_id and one for LymphGen class
+#' @param verbose Boolean, set to TRUE to print informational messages. Useful for debugging. Default is FALSE
 #'
-#' @return A data frame.
+#' @return If run with A list of data frames with the following names:
+#'  lymphgen (a data frame containing the tidy LymphGen output), 
+#'  features (a binary matrix indicating which patients had each feature), 
+#'  feature_annotation (a data frame with one row per LymphGen feature reduced to gene or arm, for arm-level events and summary statistics for the feature across the cohort), 
+#'  features_long (a data frame with one row per LymphGen feature/patient event), 
+#'  sample_annotation (a data frame with one row per sample and columns indicating the number of features for each LymphGen class in that sample)
+
 #'
-#' @import config dplyr tidyr readr stringr tibble glue GAMBLR.helpers
+#' @import config dplyr tidyr readr stringr tibble
 #' @export
 #'
 #' @examples
 #' \dontrun{
```

Comment:
Yes they run just fine, I also confirmed this! The no run directive is manly related to this function being GSC-specific so would fail if/when run in github actions